cmd/daily-sweeper.go

```diff
@@ -119,9 +164,9 @@ func dailySweeper() {
 		break
 	}
 
-	// Perform a sweep round each month
+	// Perform a sweep round regularly
 	for {
-		if time.Since(lastSweepTime) < 30*24*time.Hour {
+		if time.Since(lastSweepTime) < 6*time.Hour {
```

Comment:
Could we have a constant for this six hour interval?

Reply:
> Could we have a constant for this six hour interval?

 Fixed